### ISSUE TITLE
sv_lan in listenserver.cfg

### ISSUE THREAD

**Issue by Tiagoquix** [mastercoms/tf2cfg#635]
I'm pretty sure that sv_lan 1 being in listenserver.cfg and not being set before the server is created makes it ineffective.

A few things that suggest this to be true:

1) https://developer.valvesoftware.com/wiki/Command_Line_Options
> When set to 1, **launches** the game in LAN mode. Useful to stop players from joining your game from the Internet.

2) Creating a server using mastercomfig 9.8.1 (sv_lan only being set after loading a map):
![image](https://user-images.githubusercontent.com/30274161/162606508-13fa0cc0-a612-4cd7-b984-b86889eb15e7.png)
> Initializing Steam libraries for secure **Internet** server
> Connection to Steam servers successful.
> Public IP to Steam is XXXX.
> Lobby-based matchmaking mode not active --- Note: tf_mm_strict 1 probably only works if tf_mm_servermode is set to 1. Since servermode is 0 by default, this cvar has no effect afaik.
> VAC secure mode is activated. --- Note: VAC doesn't activate in local servers.

sv_lan set before loading a map:
![image](https://user-images.githubusercontent.com/30274161/162606643-f03e5ec0-f2c0-4902-bca4-1cb9f9088ba5.png)
> Initializing Steam libraries for **LAN** server
> Doesn't mention "Connection to Steam servers"
> Doesn't mention "Public IP to Steam"
> VAC secure mode is disabled.

In short: changes made to listenserver.cfg should be rolled back. sv_lan must be defined in comfig.cfg and tf_mm_strict doesn't seem to make a difference.

**Comment by mastercoms** [comment]
Fixed in 49b693fd017ee708d2938c758ec752c896d1d37b
